sftpgo-plugin-auth.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: sftpgo-plugin-auth
+  version: 1.0.8
+  epoch: 0
+  description: "LDAP/Active Directory authentication for SFTPGo"
+  copyright:
+    - license: AGPL-3.0-only
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - git
+      - go
+      - wolfi-baselayout
```

Comment:
As pipelines are used for everything, I think the whole environment block can be dropped. I think this applies to all the files proposed here.

Reply:
I already fixed them, please help to review @xnox